workup.rb

```diff
@@ -0,0 +1,118 @@
+require 'mixlib/shellout'
```

Comment:
Why not just turn this into a gem and install it so you can run `chef exec workup` ?